tests/test_configuration.py

```diff
@@ -515,3 +517,10 @@ def test_get_fw_version(motion_controller):
 
     assert firmware_version_0 == expected_fw_version_0
     assert firmware_version_1 == expected_fw_version_1
+
```

Comment:
You could implement a `no_connection` unit test for `get_vendor_id` function.

Reply:
Done!